schema.h

```diff
@@ -35,11 +35,26 @@ SOFTWARE.
 // Data structures for internal storage.
 enum class UID : uint64_t { INVALID = 0u };
 enum class CID : uint64_t { INVALID = 0u };
-enum class ANSWER : int { UNSEEN = 0, CTFO = 1, TFU = 2, TIFB = -1 };
+enum class ANSWER : int { UNSEEN = 0, CTFO = 1, TFU = 2, SKIP = -1 };
+enum class AUTH_TYPE : int { UNDEFINED = 0, IOS };
+
+const std::vector<unsigned int> LEVEL_SCORES{
```

Comment:
This won't pass the ODR and `make check`. If it should be in a header file, wrap it into a `Singleton`.
Better yet, have this singleton read and parse an input JSON with the mapping.